packages/react-native-editor/__device-tests__/helpers/utils.js

```diff
@@ -1,21 +1,19 @@
 /**
  * External dependencies
  */
-import childProcess from 'child_process';
+const childProcess = require( 'child_process' );
 // eslint-disable-next-line import/no-extraneous-dependencies
-import wd from 'wd';
-import crypto from 'crypto';
-import path from 'path';
-import fs from 'fs';
-
+const wd = require( 'wd' );
+const crypto = require( 'crypto' );
+const path = require( 'path' );
 /**
  * Internal dependencies
  */
-import serverConfigs from './serverConfigs';
-import { iosServer, iosLocal, android } from './caps';
-import AppiumLocal from './appium-local';
+const serverConfigs = require( './serverConfigs' );
```

Comment:
Curious why you converted import to require here and in `packages/react-native-editor/__device-tests__/pages/editor-page.js `. Feels like maybe theres a reason I'm missing.

Reply:
That's a good question! These files are now running in a different jest environment than before, and it's currently not possible to apply the same babel transforms/presets we do to the tests themselves. A PR merged to jest resolves this issue and will hopefully be released in the next versions. https://github.com/facebook/jest/pull/8751

Reply:
got it, thanks!